src/app/GitExtUtils/GitUI/Theming/ThemeFix.cs

```diff
@@ -68,12 +66,12 @@ private static void SetupTextBoxBase(TextBoxBase textBox)
 
         private static void SetupToolStrip(ToolStrip strip)
         {
-            strip.UseExtendedThemeAwareRenderer();
+            strip.RenderMode = ToolStripRenderMode.Professional;
         }
 
         private static void SetupContextMenu(ContextMenuStrip strip)
         {
-            strip.UseExtendedThemeAwareRenderer();
+            strip.RenderMode = ToolStripRenderMode.Professional;
```

Comment:
This is the actual change.
Everything else is removal of dead or empty code - which is hard to see on GH.

Reply:
Before the ToolStripExThemeAwareRenderer.cs was set, but that did not have any effect than setting the professional renderer.

Some toolstrip controls set extensions to system/professional renderers. Without that the toolstrip menues have one color background, the darker of the current:
![image](https://github.com/user-attachments/assets/407b2444-8c88-44ce-9f5e-a3da98f52452)
Set in: src/app/GitExtUtils/GitUI/ToolStripExtensions.cs
I wanted to remove that extension too, but did not see a way.

I do not believe FileStatusList.Toolbar needs further changes.